### PR TITLE
Adds broken link checker circle ci integration and fixes broken links

### PR DESCRIPTION
**Pull request by evans**

### CODE REVIEW

File: source/troubleshooting.md [hunk 1]
```diff
@@ -73,4 +73,4 @@ Please include the following when submitting an issue to our support team:
 * Platform of GraphQL server
 * The query submitted and the full response
 
-Submit your issue to [support@apollographql.com](mailto:support@apollographql.com) or join our Apollo public slack #engine channel here (https://apollographql.slack.com/).
+Submit your issue to [support@apollographql.com](mailto:support@apollographql.com) or join our Apollo public slack [#engine channel](https://apollographql.slack.com/messages/C7FFYGLQG).
```
Comment (abernix) [inline source/troubleshooting.md]: _Nitty: Generally speaking, and even though this is _similar_ to the other portion of this PR, in order to keep things moving along and as a best practice that the actual link fixes should be a separate PR. For example: I think this change is super valuable could land immediately! I think my line here is: "Should this land by itself, even if the other portion of the PR doesn't?"_

Comment (abernix) [reply source/troubleshooting.md]: My one discussion point on this might be: Could we use `engine` in place of `C7FFYGLQG` for ease-of-maintainability within the docs?

(The counterpoint to this might be "well, if we rename the channel in Slack and forget to update it here this ID ensures it still works", to which I'd probably say "You're right!" and then suggest we remove the `#engine` reference in the link itself.)

File: package.json [hunk 2]
```diff
@@ -14,13 +14,16 @@
...
-    "meteor-theme-hexo": "1.0.5"
+    "meteor-theme-hexo": "1.0.5",
+    "poke-site": "1.3.1",
+    "start-server-and-test": "1.4.1"
   },
   "scripts": {
     "start": "npm run build && chexo apollo-hexo-config -- server",
     "build": "chexo apollo-hexo-config -- generate",
     "clean": "hexo clean",
-    "test": "npm run clean; npm run build"
```
Comment (abernix) [inline package.json]: Where would the `npm run build` command get run for tests with this new setup?  For example, on a new repository, if someone cloned and ran [`npm install-test`](https://docs.npmjs.com/cli/install-test), how would the `build` script be run?  Does `poke` do that automatically?

File: package.json [hunk 3]
```diff
@@ -14,13 +14,16 @@
...
+    "ci": "npm run clean; start-server-and-test start http-get://localhost:4000 test",
```
Comment (abernix) [inline package.json]: What does this `http-get` scheme accomplish here?  I feel like introducing another web server here (this repository already has `hexo-server`) seems undesirable.  I imagine that the `hexo-server` would present the problem of not shutting down automatically, but what if we just ran on the local HTML directly?

File: source/setup-standalone.md [hunk 4]
```diff
@@ -289,7 +289,7 @@ engine.listen({
 });
 ```
 
-`start` has some optional options. Note that these are the same as the options in the `launcherOptions` option to [`ApolloEngine.listen()`](/setup-node.html#api-engine.listen)... now you know why that option has that name!
+`start` has some optional options. Note that these are the same as the options in the `launcherOptions` option to [`ApolloEngine.listen()`](./setup-node.html#api-engine.listen)... now you know why that option has that name!
```
Comment (abernix) [inline source/setup-standalone.md]: 🥇 Eeeps!  Can't wait until we actually have a link checker in place.

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  CompleteFlow:
```
Comment (abernix) [inline .circleci/config.yml]: Baring other considerations in this PR: maybe just "Docs" here instead of "Complete Flow" so it shows up as "Docs" in the status checks (with the same consistency as other deployments, too)?

File: package.json (hunk 3 above)
Comment (evans) [reply package.json]: `start-server-and-test` runs `npm start` and then waits until `localhost:4000` starts serving content, then calls `npm test`. The http-get scheme forces a GET request as opposed to a HEAD request, which hexo does not support. Running on the local files would be a nice feature! We'd have to add it to poke-site though 😞

File: package.json (hunk 2 above)
Comment (evans) [reply package.json]: True, maybe `npm test` could start the server and run the poke tests. `start-server-and-test` will allow the server startup to fail and then run the tests. We should probably ensure there is a timeout though 😄

File: .circleci/config.yml (hunk 5 above)
Comment (evans) [reply .circleci/config.yml]: Docs is much better 🤦‍♂️ Thanks!

File: source/troubleshooting.md (hunk 1 above)
Comment (evans) [reply source/troubleshooting.md]: Both points make sense! I'll split the PR

**Comment by evans** [comment]
@abernix I like the idea of integrating it higher up in our docs stack. It looks like the package you linked is from a while ago and doesn't "just work". I'll look into it in the future, maybe this weekend?

**Comment by abernix** [comment]
@evans Let's maybe do a quick pair on this next week --- 15 minutes or so; nothing too much? I have a couple ideas, but past EOD for me.

**Comment by zionts** [comment]
I'm excited for this to land :D

**Comment by zionts** [comment]
@evans is this d.e.d?

**Comment by abernix** [comment]
This probably won't land in this exact repository anyway and there's no sense in leaving a single floating PR here, so let's revisit this in the future on a higher level repository (e.g. https://github.com/meteor/meteor-theme-hexo).

I've opened https://github.com/meteor/meteor-theme-hexo/issues/73 to track it over there.

Short term solution: https://www.deadlinkchecker.com/.
